### PR TITLE
STITCH-1187 Android SDK: Create Admin SDK

### PR DESCRIPTION
**Pull request by jsflax**

### CODE REVIEW

File: stitch/build.gradle [hunk 1]
```diff
@@ -69,6 +69,10 @@ android {
         timeOutInMs 10 * 60 * 1000  // 10 minutes
         installOptions "-d","-t"
     }
+
+    packagingOptions {
+        pickFirst 'protobuf.meta'
```
Comment (edaniels) [inline stitch/build.gradle]: what did this do?

File: stitch/build.gradle [hunk 2]
```diff
@@ -114,8 +119,9 @@ dependencies {
     compile 'com.mcxiaoke.volley:library:1.0.19'
 }
 
-if (rootProject.file('local.properties').exists()) {
-    apply from: 'https://raw.githubusercontent.com/nuuneoi/JCenter/master/installv1.gradle'
-    apply from: 'https://gist.githubusercontent.com/edaniels/c9e9da05491cfda34151a9275df3eb2f/raw/4acaa3de0ba08ea62d0e05cd1803e044e1bdb80f/bintrayv1.gradle'
-}
+
+//if (rootProject.file('local.properties').exists()) {
```
Comment (edaniels) [inline stitch/build.gradle]: Why comment this out?

File: stitch/build.gradle (hunk 1 above)
Comment (jsflax) [reply stitch/build.gradle]: Seems to have to do with the latest version of espresso: https://issuetracker.google.com/issues/69333598

File: stitch/build.gradle (hunk 2 above)
Comment (jsflax) [reply stitch/build.gradle]: Uncommenting.

File: stitch/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -3,6 +3,7 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="com.google.android.c2dm.permission.RECEIVE" />
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
```
[the remark this answers is not in the document]
Comment (jsflax) [reply stitch/src/main/AndroidManifest.xml]: Removing.

File: .idea/misc.xml [hunk 4]
```diff
@@ -24,7 +24,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
[the remark this answers is not in the document]
Comment (jsflax) [reply .idea/misc.xml]: :sigh:

File: stitch/src/androidTest/java/com/mongodb/stitch/TestUtils.kt [hunk 5]
```diff
@@ -98,3 +107,35 @@ fun mockResponseBuilder(body: String,
     mockResponse.block()
     return mockResponse
 }
+
+@RunWith(AndroidJUnit4::class)
+open class StitchTestCase {
+    internal val instrumentationCtx: Context by lazy { InstrumentationRegistry.getContext() }
```
[the remark this answers is not in the document]
Comment (jsflax) [reply stitch/src/androidTest/java/com/mongodb/stitch/TestUtils.kt]: It's Diet Java with a hint of Swift.

**Pull request merged**